conf/brain.config

```diff
@@ -0,0 +1,72 @@
+// ─────────────────────────────────────────────────────────────────────────────
+// Global Parameters
+// ─────────────────────────────────────────────────────────────────────────────
+params {
+    config_profile_name        = 'Brain Image Library Analysis Ecosystem'
+    config_profile_description = 'A public resource enabling researchers to deposit, analyze, mine, share, and interact with large brain image datasets.'
+    config_profile_url         = 'https://www.brainimagelibrary.org'
+    config_profile_contact     = 'Ivan Cao-Berg (@icaoberg)'
+
+    // General resource boundaries (used in profile as well)
+    max_memory = 2.9.Tb
+    max_cpus   = 80
+    max_time   = 48.h
+
+    // Ignore iGenomes when not needed
+    igenomes_ignore = true
+    igenomesIgnore  = true
+}
+
+// ─────────────────────────────────────────────────────────────────────────────
+// Singularity Configuration
+// ─────────────────────────────────────────────────────────────────────────────
+singularity {
+    enabled     = true
+    autoMounts  = true
+    runOptions  = '''
+        --home $HOME:/bil/users/$USER \
+        --bind /scratch \
+        --bind /local
+    '''
+}
+
+// ─────────────────────────────────────────────────────────────────────────────
+// Executor Configuration
+// ─────────────────────────────────────────────────────────────────────────────
+executor {
+    queueSize       = 10
+    submitRateLimit = '5 sec'
+}
+
+// ─────────────────────────────────────────────────────────────────────────────
+// Profiles
+// ─────────────────────────────────────────────────────────────────────────────
+profiles {
+    compute {
+        params {
+            config_profile_name        = 'BIL - General Computing Partition'
+            config_profile_description = 'Brain Image Library Analysis Ecosystem at the Pittsburgh Supercomputing Center'
+            config_profile_url         = 'https://www.brainimagelibrary.org'
+            config_profile_contact     = 'Ivan Cao-Berg (@icaoberg)'
+        }
+
+        process {
+            executor        = 'slurm'
+            queue           = 'compute'
+            queueSize       = 10
+            clusterOptions  = '--qos=normal'
+
+            // Set scratch directory to /local
+            scratch        = true
+            scratchMount   = '/local'
+
+            // Default per-process limits
+            resourceLimits = [
+                memory: 128.Gb,
+                cpus  : 16,
+                time  : 4.h
+            ]
+        }
+    }
+}
+
```

Comment:
You may want to consider the `cleanup` option: https://www.nextflow.io/docs/latest/reference/config.html#unscoped-options to reduce HDD space after successful completion.